src/Browser/Avalonia.Browser/webapp/modules/avalonia/input.ts

```diff
@@ -171,11 +171,13 @@ export class InputHelper {
         const pointerDownHandler = (args: PointerEvent) => {
             pointerDownCallback(args);
             args.preventDefault();
+            element.setPointerCapture(args.pointerId);
         };
 
         const pointerUpHandler = (args: PointerEvent) => {
             pointerUpCallback(args);
             args.preventDefault();
+            element.releasePointerCapture(args.pointerId);
```

Comment:
Can you try to implement a custom IPointer?
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Input/IPointer.cs#L29

We have something like that for Windows (and only windows):
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Windows/Avalonia.Win32/Input/WindowsMouseDevice.cs#L24-L44

Nuance about Windows implementation is that any touch and pen input is automatically captured on pointer press/click. But WM_MOUSE input is not automatically captured, so we do it manually there with custom IPointer.

If it's the same on Browser, it would make sense to do similarly. But should be tested on different devices.

Reply:
The problem is that we are not passing pointer id there.
It's going to be a bit more complicated, but in general we already pass RawPointerId to the pointer args:
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Browser/Avalonia.Browser/BrowserTopLevelImpl.cs#L95-L103

It would require a second overload here: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Input/IPointer.cs#L29
```
interface IPointer
{
    void Capture(IInputElement? control);
    void Capture(IInputElement? control, PointerEventArgs pointerArgs);
}
```

So, yeah, not sure if it worth doing these changes. We can keep this PR as it is.